html/partials/os.html.haml

```diff
@@ -0,0 +1,25 @@
+- if os
+  %div
+    %a.scope_anchor{:id => "os"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace